projects/igniteui-angular/src/lib/core/styles/components/input/_input-group-theme.scss

```diff
@@ -910,6 +909,10 @@
             line-height: normal; /* Fix placeholder position in Safari */
         }
     }
+    
+    input:not([type='date']) {
```

Comment:
Mm, actually, I think it's better if we scope this to the %form-group-input only.